lib/bundler/lockfile_generator.rb

```diff
@@ -86,7 +86,7 @@ def add_section(name, value)
           out << "  #{key}: #{val}\n"
         end
       when String
-        out << "   #{value}\n"
```

Comment:
This is causing the ```BUNDLED_WITH``` and ```RUBY_VERSION``` sections of the lockfile to have an extra space in from of each line of content.